modules/bookManager.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
- [ ] As the previous CR instructed please avoid disabling linters, they help make your code work better en make it easier to read.